Brainduino/Brain.cpp

```diff
@@ -9,26 +9,26 @@
 
 Brain::Brain(Print *printer, Stream *stream_in, BrainDelegate *delegate)
 {
-    set(printer, stream_in, delegate, nullptr);
+    set(printer, stream_in, delegate, 0);
```

Comment:
Now it is not code anymore, because we have the method `setCode`... This parameter now calls `seed` and it is the `randomSeed`. It is an integer btw